runtime/lua/vim/lsp.lua

```diff
@@ -2144,11 +2145,8 @@ end
 ---@param bufnr (integer) Buffer handle, or 0 for current.
 ---@param method (string) LSP method name
 ---@param params (table|nil) Parameters to send to the server
----@param handler fun(results: table<integer, {error: lsp.ResponseError, result: any}>) (function)
---- Handler called after all requests are completed. Server results are passed as
---- a `client_id:result` map.
```

Comment:
we lost information here, why were the sentences deleted? |lsp-handler| doesn't give the extra insight that is specific to `buf_request_all` about "all requests" and "client_id:result map".

Reply:
just use `lsp-handler` is enough we have a `@alias` for it.

Reply:
It's good to avoid redundant docs, but my comment above explained why information was lost. Linking to `|lsp-handler|` loses the extra content that is specific to `buf_request_all`, which accepts _multiple_ results, thus the handler looks different.

Reply:
I think i get your point  .